app/org/sagebionetworks/bridge/play/controllers/AuthenticationController.java

```diff
@@ -98,9 +98,8 @@ public Result resendEmailVerification() throws Exception {
     }
 
     public Result requestResetPassword() throws Exception {
-        JsonNode json = requestToJSON(request());
         Email email = parseJson(request(), Email.class);
-        Study study = getStudyOrThrowException(json);
+        Study study = studyService.getStudy(email.getStudyIdentifier());
```

Comment:
Yes there's no test. Added a couple for requestResetPassword, added back the app version kill switch check.